android/src/main/kotlin/eu/nk2/tinkoff_acquiring_sdk/TinkoffProtocol.kt

```diff
@@ -85,10 +142,38 @@ fun TinkoffFeaturesOptions.toTinkoff(): FeaturesOptions {
     if(enableCameraCardScanner) options.cameraCardScanner = CameraCardIOScanner()
     options.darkThemeMode = darkThemeMode
     options.localizationSource = AsdkSource(language)
+    options.emailRequired = emailRequired
+
 
     return options
 }
 
+fun TinkoffShop.toTinkoff(): Shop {
+    return Shop(this.shopCode ?: "", this.name ?: "", this.amount ?: 0, this.fee)
```

Comment:
Default parameters generally should be handled by throwing errors if passed as nulls